pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[project]
+name = "khal"
+dynamic = ["version"]
+description = "A standards based terminal calendar"
+readme = "README.rst"
+authors = [
+  {name = "khal contributors", email = "khal@lostpackets.de"},
```

Comment:
**tangential and non-blocking**: what do you think of simply moving `AUTHORS` into this array?